dsc-util.sh

```diff
@@ -46,6 +46,10 @@ main_menu()
     get-logs)
       get_logs_func
       ;;
+    pull_latest)
+      check_login
+      podman pull $REG_PATH/$LATEST_IMAGE
```

Comment:
Hello @takaelucky 

Is this the only step necessary to update the container to the latest version?

Thank you!
Waldirio